packages/lib-classifier/src/store/SubjectViewerStore.js

```diff
@@ -73,6 +73,7 @@ const SubjectViewer = types
       onError (error) {
         console.error(error)
         self.loadingState = asyncStates.error
+        return error
```

Comment:
```suggestion
        error
```

Reply:
I don't think this returned value gets used anywhere.

Reply:
Ok, I think this can be removed entirely then. I was misunderstanding your consistent return suggestion and thought you meant that the error should be returned.